web/src/components/MultiSelect.jsx

```diff
@@ -23,21 +24,33 @@ export default function MultiSelect({ className, title, options, selection, onTo
       </div>
       {state.showMenu ? (
         <Menu relativeTo={popupRef} onDismiss={() => setState({ showMenu: false })}>
-          <Heading className="p-4 justify-center" size="md">{title}</Heading>
+          <div className="flex flex-wrap gap-2 items-center">
+            <Heading className="p-4 justify-center" size="md">{title}</Heading>
+            <Button className="mx-2" onClick={() => onShowAll() }>
+              Show All
+            </Button>
+          </div>
           {options.map((item) => (
-            <label
-              className={`flex flex-shrink space-x-2 p-1 my-1 min-w-[176px] hover:bg-gray-200 dark:hover:bg-gray-800 dark:hover:text-white cursor-pointer capitalize text-sm`}
-              key={item}>
-              <input
-                className="mx-4 m-0 align-middle"
-                type="checkbox"
-                checked={selection == "all" || selection.indexOf(item) > -1}
-                onChange={() => onToggle(item)} />
-              {item.replaceAll("_", " ")}
-            </label>
+            <div className='flex grow'>
+              <label
+                className={`flex flex-shrink space-x-2 p-1 my-1 min-w-[176px] hover:bg-gray-200 dark:hover:bg-gray-800 dark:hover:text-white cursor-pointer capitalize text-sm`}
+                key={item}>
+                <input
+                  className="mx-4 m-0 align-middle"
+                  type="checkbox"
+                  checked={selection == "all" || selection.split(',').indexOf(item) > -1}
+                  onChange={() => onToggle(item)} />
+                {item.replaceAll("_", " ")}
+              </label>
+              <div className=' justify-right'>
+                <Button className="mx-2" onClick={() => onSolo(item)}>
+                  👀
```

Comment:
This is funny, but I think it'd be better to use an icon like the radio button icon or something simpler.

Reply:
https://user-images.githubusercontent.com/57186372/208248034-af3358bc-f9fd-476f-895d-79f480adb62b.mp4

wat u think ?

Reply:
I think that looks great. 

One other thing I noticed (just got to my computer and built this branch), it seems like when you open the list of filters the `Show All` button is focused by default which isn't ideal

Reply:
> Looks good, did you have a chance to look at the button autofocus issue? I have a screen cap showing it:
> Screen.Recording.2022-12-17.at.11.13.24.AM.mov
> 
> I think the fix may be as simple as just setting `autofocus` on the main div element

Looks like autofocus for MultiSelect menu is done in https://github.com/blakeblackshear/frigate/blob/9b99ba81e506d923487191b6763a83c0d41a7669/web/src/components/RelativeModal.jsx#L96-L97

I set ShowAll button tabindex so it will nolonger be autofocus by default.